### ISSUE TITLE
Error " request failed with status code 400 " raised when trying to import an Excel file 

### ISSUE THREAD

**Issue by okaeiz** [nocodb/nocodb#1313]
- Project Info

Node: **v12.22.6**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **mysql2**
ProjectOnRootDB: **true**
RootDB: **mysql2**
PackageVersion: **0.84.12**

- Steps To Reproduce
When creating a new project using an Excel file, error " request failed with status code 400 " is raised. The column headers are visible but we cannot see the rest of the data ( rows haven't been imported ).
We cannot also see the Kanban view and Calendar view of our databases. 

- Expected behavior
We expected to have our Excel file completely imported; with all rows and columns. 

![Screenshot (2)](https://user-images.githubusercontent.com/100111649/154900274-9afb6b3b-a99a-4b3d-b353-af92dda8ea3e.png)

Join our discord : https://discord.gg/5RgZmkW for realtime help.

Node version : **v12.22.1**
Arch type: **x64**
Platform: **linux**
Docker: **true**
Database: **sqlite3**
packageVersion: **0.9.27**

**Comment by wingkwong** [comment]
Able to reproduce only in docker-compose

**Comment by dstala** [comment]
We cannot also see the Kanban view and Calendar view of our databases.

>> Right now, Kanban & Calendar views are under development, not deployed yet. Grid/ Gallery & Form view are options available.

**Comment by wingkwong** [comment]
Checked with the user. The excel file contains data with Persian characters. By default, mysql is using ``latin1_swedish_ci``.  It can be proved by running the following SQL query.

```
SELECT SCHEMA_NAME 'database', default_character_set_name 'charset', DEFAULT_COLLATION_NAME 'collation' FROM information_schema.SCHEMATA;
```

You should see

```
+--------------------+---------+-------------------+
| database           | charset | collation         |
+--------------------+---------+-------------------+
| information_schema | utf8    | utf8_general_ci   |
| root_db            | latin1  | latin1_swedish_ci |
+--------------------+---------+-------------------+
```

Hence, adding the following line will change the character set and collation.

```
command: --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci
```

Example:

![image](https://user-images.githubusercontent.com/35857179/154922399-0aea74a5-1f7d-4850-ba4c-21093a3bbf8a.png)

Running the same query

```
SELECT SCHEMA_NAME 'database', default_character_set_name 'charset', DEFAULT_COLLATION_NAME 'collation' FROM information_schema.SCHEMATA;
```

Result:

```
+--------------------+---------+--------------------+
| database           | charset | collation          |
+--------------------+---------+--------------------+
| information_schema | utf8    | utf8_general_ci    |
| root_db            | utf8mb4 | utf8mb4_unicode_ci |
+--------------------+---------+--------------------+
```

**Comment by wingkwong** [comment]
Included an example in https://github.com/nocodb/nocodb/commit/a9481ddadb347b66dfc3d76f7f66aa655c5d019f.

**Comment by adamlaska** [comment]
> Fixed in ``v0.84.13``.
